Dockerfile

```diff
@@ -18,29 +20,28 @@ RUN \
   apk add --no-cache \
     fontconfig \
     memcached \
-    php8-ctype \
-    php8-curl \
-    php8-dom \
-    php8-gd \
-    php8-ldap \
-    php8-mbstring \
-    php8-mysqlnd \
-    php8-openssl \
-    php8-pdo_mysql \
-    php8-pecl-memcached \
-    php8-phar \
-    php8-simplexml \
-    php8-tokenizer \
+    php81-ctype \
+    php81-curl \
+    php81-dom \
+    php81-gd \
+    php81-ldap \
+    php81-mbstring \
```

Comment:
On the base

Reply:
Resolved in https://github.com/linuxserver/docker-bookstack/pull/158/commits/1603fb13fcf14f0120b03b29dc13a45f0d5d69f5